src/javascript/static/pages/tnc.js

```diff
@@ -22,6 +23,17 @@ const TermsAndConditions = (() => {
         window.onresize = checkWidth;
 
         $('.currentYear').text(new Date().getFullYear());
+
+        handleContactNumber();
+    };
+
+    const handleContactNumber = () => {
+        const current_loginid = LocalStore.get('active_loginid');
+        
+        if (current_loginid && current_loginid.startsWith('MLT')) {
```

Comment:
same here, please don't check broker code, we check landing company instead

Reply:
In my new commit, I used `landing company` instead of `loginid`